README.rst

```diff
@@ -101,6 +123,10 @@ The official documentation is hosted on SourceForge
 
     http://statsmodels.sf.net/
 
+or Github
+
+    http://statsmodels.github.io/dev/
+
 
 Windows Help
 ============
```

Comment:
delete this entire `Windows Help` section. I haven't updated and used it in a long time.